go-controller/pkg/libovsdb/libovsdb.go

```diff
@@ -191,6 +193,22 @@ func newSSLKeyPairWatcherFunc(certFile, privKeyFile string, tlsConfig *tls.Confi
 			select {
 			case event, ok := <-watcher.Events:
 				if ok && event.Op&(fsnotify.Write|fsnotify.Remove) != 0 {
+					if event.Op&fsnotify.Remove != 0 {
+						// cert/key file removed, need wait for the file to be created again.
+						if err := wait.PollImmediate(10*time.Millisecond, 5*time.Second, func() (bool, error) {
+							if _, err := os.Stat(event.Name); os.IsNotExist(err) {
+								return false, nil
+							}
+							return true, nil
+						}); err != nil {
+							klog.Errorf("Fatal error: tiemout waiting for %s to be created", event.Name)
+							os.Exit(1)
+						}
+						if err := watcher.Add(event.Name); err != nil {
```

Comment:
as @dcbw pointed out to me, there is probably no reason to add back the event to watch the same file right?